webapp/src/components/ParcelAttributes/ParcelAttributes.js

```diff
@@ -1,42 +1,65 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Link } from 'react-router-dom'
+import { Icon } from 'semantic-ui-react'
+
 import { parcelType } from 'components/types'
-import ParcelTags from 'components/ParcelTags/ParcelTags'
-import Icon from 'components/Icon'
 import './ParcelAttributes.css'
-import { locations } from 'locations'
 
 export default class ParcelAttributes extends React.PureComponent {
   static propTypes = {
     parcel: parcelType,
-    withLink: PropTypes.bool,
-    withTags: PropTypes.bool
+    status: PropTypes.string,
+    onClick: PropTypes.func,
+    onDelete: PropTypes.func
   }
 
   static defaultProps = {
-    withLink: true,
-    withTags: true
+    status: ''
   }
 
-  render() {
-    const { parcel, withLink, withTags } = this.props
+  handleOnClick = () => {
+    const { onClick, parcel } = this.props
+    if (onClick) {
+      onClick(parcel)
+    }
+  }
+
+  handleOnDelete = event => {
+    const { onDelete, parcel } = this.props
+    onDelete(parcel)
+    event.stopPropagation()
+    event.nativeEvent.stopImmediatePropagation()
+  }
+
+  getClassName() {
+    const { status, onClick, onDelete } = this.props
+    const statusClass = status
+      ? `has-status status-${status.toLowerCase()}`
+      : ''
+    const onClickClass = onClick ? 'clickeable' : ''
+    const onDeleteClass = onDelete ? 'deleteable' : ''
+    return `ParcelAttributes ${statusClass} ${onClickClass} ${onDeleteClass}`
+  }
 
-    const Wrapper = withLink ? Link : React.Fragment
```

Comment:
Nope! instead of adding a fixed link to the attributes, I added an `onClick` attribute, and the EstateDetail has a `onClick={parcel => push(locations.parcelDetail(parcel.x, parcel.y))}` (not real code)
That way the code it's more flexible and accomplishes the same result. For example the auction needs to re-center the map instead of redirecting